source/changelog.txt

```diff
@@ -12,6 +12,48 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.3.7
+------
+
+*Released December 17, 2024*
+
+Bug Fixes:
+
+- :issue:`MONGOSH-1943` - Only ever run ``MongoshRepl.onExit`` once
+- :issue:`MONGOSH-1969` - Don't override output that came from onPrint
+
+`Full release notes available on JIRA
+<https://jira.mongodb.org/browse/MONGOSH-1922?jql=project%20%3D%20MONGOSH%20AND%20fixVersion%20%3D%202.3.7>`__.
+
+v2.3.6
+------
+
+*Released December 13, 2024*
+
+Bug Fixes:
+
+- :issue:`MONGOSH-1943` - Account for possibility of process.exit() throwing 
+  under coverage; Improve debuggability of ``mongosh not initialized yet`` errors
+- :issue:`COMPASS-8576` - Keep operation in progress while switching tabs 
+
+`Full release notes available on JIRA
+<https://jira.mongodb.org/browse/MONGOSH-1922?jql=project%20%3D%20MONGOSH%20AND%20fixVersion%20%3D%202.3.6>`__.
```

Comment:
As the release notes ticket says, this is really a fully internal release. None of these changes are user-facing, and I'd say as much here.

```suggestion
Contains internal enhancements and improvements.

`Full release notes available on JIRA
<https://jira.mongodb.org/browse/MONGOSH-1922?jql=project%20%3D%20MONGOSH%20AND%20fixVersion%20%3D%202.3.6>`__.
```

(This is taken from the phrasing we use for other releases of this type)